### PR TITLE
Make help panel contents customisable

### PR DESCRIPTION
**Pull request by reginapramesti**
### What this PR does

Fixes https://github.com/TerriaJS/terriajs/issues/4161 and https://github.com/TerriaJS/terriajs/issues/4182

Made some changes to the way the help panel is implemented. This is so that the help panel contents are more modular, and could be individually customised in every map

### Checklist

-   [x] There are unit tests to verify my changes are correct.
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by reginapramesti** [comment]
Things to unit test:
- [x] The number of help items equals the number of items listed on config
- [x] Providing no content will not add any text
- [x] Some markdown text content will produce x number of items on the video panel
- [x] Providing no icon will not break, instead use default icon
- [x] Providing x icon will use that icon if exists in terria
- [x] Providing no video and/or placeholder images will not break, just don't show video
- [x] Providing video and placeholder images -> expect a video component

File: test/ReactViews/Map/Panels/HelpPanel/HelpPanelSpec.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    it("renders 1 help menu item", function() {
+      const helpItems = testRenderer.root.findAllByType(HelpPanelItem);
+      expect(helpItems.length).toBe(1);
+    });
+
+    it("renders the default start icon", function() {
+      const helpItem = testRenderer.root.findByType(HelpPanelItem);
+      const menuIcon = helpItem.findByType(Icon);
+      // Not sure how to compare icons so I just used the gylph.id
```
Comment (soyarsauce) [inline test/ReactViews/Map/Panels/HelpPanel/HelpPanelSpec.tsx:74]: this is fine

**Review by soyarsauce — CHANGES_REQUESTED** [review]
Hi Regina - this looks really really awesome!
Could you please open a PR for TerriaMap to include the intended default panels and content that we would ship by default?
Really good work with some basic check-if-it-renders tests, too!

**Comment by soyarsauce** [comment]
Once you add the terriamap PR, merge this + the terriamap-to-mobx together and :shipit:

File: lib/ReactViews/Map/Panels/HelpPanel/StyledHtml.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import Text from "../../../../Styled/Text";
+import Box from "../../../../Styled/Box";
+import styled from "styled-components";
+
+const Numbers = styled(Text)`
+  width: 22px;
+  height: 22px;
+  line-height: 22px;
+  border-radius: 50%;
+  ${props => props.darkBg && `background-color: ${props.theme.textDarker};`}
```
[the remark this answers is not in the document]
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/StyledHtml.jsx]: Yup that's true, will do that then

**Comment by reginapramesti** [comment]
TerriaMap PR created! https://github.com/TerriaJS/TerriaMap/pull/445

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx [hunk 3]
```diff
@@ -48,11 +40,24 @@ class HelpPanelItem extends React.Component {
...
-      this.props.viewState.selectedHelpMenuItem === this.props.itemString;
+      this.props.viewState.selectedHelpMenuItem === this.props.content.itemName;
     const className = classNames({
       [Styles.panelItem]: true,
       [Styles.isSelected]: itemSelected
     });
+
+    const iconName = this.props.content.icon
+      ? this.props.content.icon
+      : "start"; // TODO: check what default icon should be
```
[the remark this answers is not in the document]
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: Oops I completely forgot to do this, but now it's replaced with the video icon (which I believe is more appropriate?)

**Pull request merged**